src/modules/users/users.conf

```diff
@@ -145,47 +144,27 @@ doAutologin:     true
 # To require specific password characteristics:
 #  - set 'allowWeakPasswords' to false (the default)
 #  - configure password-checking, e.g. with NIST settings
+#
+# The defaults follow NIST-800-63B guidelines.
```

Comment:
Please keep the link to this document (which you removed lower down) because NIST doesn't necessarily mean anything to distro-makers.